numpy/random/tests/test_generator_mt19937.py

```diff
@@ -1054,6 +1063,29 @@ def test_dirichlet_alpha_non_contiguous(self):
                                   size=(3, 2))
         assert_array_almost_equal(non_contig, contig)
 
+    def test_dirichlet_small_alpha(self):
+        eps = 1.0e-9  # 1.0e-10 -> runtime x 10; 1e-11 -> runtime x 200, etc.
+        alpha = eps * np.array([1., 1.0e-3])
+        random = Generator(MT19937(self.seed))
+        actual = random.dirichlet(alpha, size=(3, 2))
+        expected = np.array([
```

Comment:
I appreciate adding whitespace for readability, but I think the space in this array is too much. Maybe something like

```
        expected = np.array([
            [[1., 0.],
             [1., 0.]],
            [[1., 0.],
             [1., 0.]],
            [[1., 0.],
             [1., 0.]]
        ])

Reply:
Fixed according to your suggestions.